backend/routes/movies.js

```diff
@@ -13,10 +13,13 @@ movieRoute.get('/:imdb_code', async (req, res, next) => {
 
 movieRoute.get('/', middleware.authRequired, movieController.getMovieList);
 
+// TO DO do we need this route?
```

Comment:
I don't think we do, I suggest we remove it to avoid confusion. Can be done in the next commits, don't need to do it right now